examples/labs/inverse_kinematics.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# -------------------------------------------------------------------------- #
+# ----------------- INVERSE KINEMATICS WITH THESEUS AND TORCHLIE ----------- #
+# -------------------------------------------------------------------------- #
+# In this example we show how to do inverse kinematics to achieve a target
+# pose for a specific end effector of a robot described in a URDF file.
+# We show two ways to accomplish this:
+# 1) solve IK with body/spatial jacobian
+# 2) solve IK as NLS optimization
+import os
+
+import torch
+
+import theseus as th
+from lie.functional import SE3 as SE3_Func
+from theseus.labs.embodied.robot.forward_kinematics import (
+    Robot,
+    get_forward_kinematics_fns,
+)
+
+dtype = torch.float64
+
+# First we load the URDF file describing the robot and create a `Robot` object to
+# represent it in Python. The `Robot` class can be used to build a kinematics tree
+# of the robot.
+URDF_REL_PATH = (
+    "../../tests/theseus_tests/embodied/kinematics/data/panda_no_gripper.urdf"
+)
+urdf_path = os.path.join(os.path.dirname(__file__), URDF_REL_PATH)
+robot = Robot.from_urdf_file(urdf_path, dtype)
+
+# We can get differentiable forward kinematics functions for specific links
+# by using `get_forward_kinematics_fns`. This function creates three differentiable
+# functions for evaluating forward kinematics, body jacobian and spatial jacobian of
+# the selected links, in that order. The return types of these functions are as
+# follows:
+#
+# - fk: return poses of selected links
```

Comment:
I think we should be more explicit here, this description doesn't help the reader understand the API. 

For example, `fk` returns N + 1 outputs, where N is the number of selected links. The first N correspond to poses (tensors, I think?). And I think the last output is a list containing poses for all links? It would be really useful to include this level of detail.

Same goes for the jacobians.

Reply:
No, `fk` will only return the tuple of selected link poses.

Reply:
Ah, my bad, I missed [this](https://github.com/facebookresearch/theseus/blob/main/theseus/labs/embodied/robot/forward_kinematics.py#L143) slicing for `fk` function. Sorry about that. You should still mention that the return is a tuple of pose tensors, one for each selected link, in the same order as the link names given as input. 

Also, can you reword the jacobians part so it says "returns a tuple where the first element is a list with the body jacobians of selected links, and the second element is a tuple with the corresponding poses."?